aws/data_source_aws_launch_template.go

```diff
@@ -314,6 +314,10 @@ func dataSourceAwsLaunchTemplate() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"host_resource_group_arn": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
Since this attribute in the data source is not meant to be configurable, we should set the schema to read-only via `Computed`:

```suggestion
							Computed: true,
```